e2e/utils/components/promotion-carousel.component.ts

```diff
@@ -0,0 +1,36 @@
+import { CAROUSEL_CONTAINER, CAROUSEL_SLIDE_NAVIGATION } from '$lib/constants/test-ids.constants';
+import { Locator, Page } from '@playwright/test';
+
+export class PromotionCarousel {
+	#page: Page;
+	#container: Locator;
```

Comment:
You are right, I left them in there because we are eventually going to write tests just for the carousel. So currently unused, but gonna be used later on.